### ISSUE TITLE
Notification contents hidden even when setting unchecked

### ISSUE THREAD

**Issue by LordMethenor** [OrangeDrangon/android-messages-desktop#383]
**Describe the bug**
Notifications show as:

**Android Messages**
New Notification

when _Settings > Hide Notification Content is unchecked.

**To Reproduce**
1. Go to alt->settings
3. Uncheck 'Hide Notification Content'
4. Click in another window, cover window, or minimize to taskbar or tray (change focus)
5. Receive notification
6. Watch toast notification or win-n to open notification center

**Expected behavior**
A notification containing the sender and start/content of their message appears.

**Screenshots**
In notification area:
![Screenshot (17)](https://user-images.githubusercontent.com/37001137/167262769-dbb2cdb9-7b2e-450f-a5fb-8d1f95b4e872.png)
On desktop (As a toast):
![Screenshot (18)](https://user-images.githubusercontent.com/37001137/167262772-83fafaf5-21a4-4769-b192-8ce3a62c53aa.png)

**Desktop (please complete the following information):**
 - OS: Windows 11 Home
 - Version 21H2 Build 22000.652

**Additional context**
The received notification is not the same one received with 'Hide Notification Content' enabled.

**Comment by OrangeDrangon** [comment]
I do not think this is a notification generated by my code. The overriding of the message when hide notification content is enable is `New Message` not `New notification`. From a quick grepping `New notification` does not appear in the repository at all.
![image](https://user-images.githubusercontent.com/25403517/167268920-4447b240-5db6-422a-9e0b-7a69ede6513b.png)

**Comment by LordMethenor** [comment]
Hmm. As I stated, this isn't the notification that appears when 'Hide Notification Content' is enabled. This might be on Google, but could it be the generic notification sent by Electron apps if there's some problem with the request?

**Comment by OrangeDrangon** [comment]
Interesting. I will look into this more. Z
